jsk_arc2017_baxter/euslisp/pick-main.l

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env roseus
+
+(ros::roseus "robot_main")
+
+(require "package://jsk_arc2017_baxter/euslisp/lib/apc-interface.l")
+
+(defun pick-init (&optional (ctype :default-controller) (calib-pressure t))
+  (jsk_arc2017_baxter::apc-init :ctype ctype)
+  (send *ri* :gripper-servo-on)
+  (send *ri* :angle-vector (send *baxter* :fold-pose-back))
+  (send *ri* :wait-interpolation)
+  (if calib-pressure
+    (send *ri* :calib-pressure-threshold) nil)
+  (objects (list *baxter*))
+  t)
+
+(defun pick-mainloop (arm)
```

Comment:
Is that possible to place this function **cleanly** into the `apc-interface`? Or `pick-interface` inheriting `apc-interface` if you need.